src/Tribe/Main.php

```diff
@@ -202,6 +202,10 @@ public function bind_implementations() {
 		tribe_singleton( 'tickets.rsvp', new Tribe__Tickets__RSVP );
 		tribe_singleton( 'tickets.commerce.currency', 'Tribe__Tickets__Commerce__Currency', array( 'hook' ) );
 		tribe_singleton( 'tickets.commerce.paypal', new Tribe__Tickets__Commerce__PayPal__Main );
+
+		// REST API v1
+		tribe_singleton( 'tickets.rest-v1.main', 'Tribe__Tickets__REST__V1__Main', array( 'bind_implementations', 'hook' ) );
```

Comment:
I tried to change over to this and now it is fatals all over the place so I reverted that coding. I do not believe the end of a project is the place to make such a large change to how everything is loaded. Also, since this is based off TEC REST I believe both of these should be done at the same time to keep the coding within the same standards.